### DISCUSSION TITLE
사용자 기기의 해상도 정보를 가져올 수 있나요?

### DISCUSSION THREAD

**Discussion by HJ3150** [naverz/zepeto-studio-kor#856] [5. Q&A]
GUI 위치를 사용자 기기에 맞게 설정하고 싶은데,  사용자 기기의 해상도 정보를 가져올 방법이 있나요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 유니티와 동일하게 Screen.width, Screen.height 를 사용해서 가져올 수 있습니다.

import { ZepetoScriptBehaviour } from 'ZEPETO.Script'
import {Screen} from "UnityEngine";

export default class Resolution extends ZepetoScriptBehaviour {
    
    Start() {    
        console.log(Screen.width, Screen.height);
    }

}

**Reply by HJ3150** [reply]
감사합니다
